build.sbt

```diff
@@ -8,14 +8,14 @@ import bijection._
 val finagleVersion210 = "6.35.0"
 val finagleVersion = "6.41.0"
 
-val scalatestVersion = "3.0.1"
-val scalacheckVersion = "1.13.4"
+val scalatestVersion = "3.0.8"
+val scalacheckVersion = "1.14.0"
 
 val utilVersion210 = "6.34.0"
-val utilVersion = "6.39.0"
+val utilVersion = "19.9.0-SNAPSHOT"
```

Comment:
https://mvnrepository.com/artifact/com.twitter/util-core
Looks like 19.9.0 is out.